src/wizard/obtain_source.jl

```diff
@@ -361,4 +385,5 @@ function step2(state::WizardState)
     obtain_source(state)
     obtain_binary_deps(state)
     get_name_and_version(state)
+    get_compilers(state)
```

Comment:
Because you have changed this, you will need to go into the tests and add simulated keystrokes to select the desired compilers in all "step 2" tests.